sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -388,4 +388,16 @@ class QueryExecutionErrorsSuite extends QueryTest
             |""".stripMargin)
     }
   }
+
+  test("INVALID_FRACTION_OF_SECOND: make_timestamp failure - the fraction of sec must be zero") {
```

Comment:
let's not add so much details in the title:
```suggestion
  test("INVALID_FRACTION_OF_SECOND: in the function make_timestamp") {
```